improver/calibration/ensemble_calibration.py

```diff
@@ -1122,24 +1082,11 @@ def process(self, current_forecast, coefficients_cube, landsea_mask=None):
 
         # Use a mask to confine calibration to regions in which the mask=1.
         if landsea_mask:
-            # Assume that the ensemble mean and the ensemble variance provide
-            # an estimate of the uncalibrated location and scale parameter.
-            # TODO: Improve the handling of the uncalibrated region, so that
-            # the ensemble mean and ensemble variance are not assumed to be
-            # the location and scale parameter, respectively.
-            uncalibrated_location_parameter = (
-                self.current_forecast.collapsed(
-                    "realization", iris.analysis.MEAN))
-            uncalibrated_scale_parameter = (
-                self.current_forecast.collapsed(
-                    "realization", iris.analysis.VARIANCE))
-            self._merge_calibrated_and_uncalibrated_regions(
-                uncalibrated_location_parameter.data,
-                location_parameter_cube.data,
-                landsea_mask.data)
-            self._merge_calibrated_and_uncalibrated_regions(
-                uncalibrated_scale_parameter.data,
-                scale_parameter_cube.data,
-                landsea_mask.data)
+            # Calibration is applied to all grid points, but the areas
```

Comment:
I think the docstring for what happens if the land-sea mask is provided needs updating.

Reply:
Done